recipe/meta.yaml

```diff
@@ -7,19 +7,22 @@ package:
 
 source:
   - url: https://github.com/wjakob/nanobind/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: 2e70727484edb7a8645d26f6a9f67352a668657c34de7a603bf9c68e5cbf8ff9
+    sha256: c37c53c60ada5fe1c956e24bd4b83af669a2309bf952bd251f36a7d2fa3bacf0
   - url: https://github.com/Tessil/robin-map/archive/{{ robin_map_commit }}.tar.gz
     sha256: 2f4be670fa4f53c3261ed7af392b414a00e75591f87da0a8dd525de376430747
     folder: ext/robin_map
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+  number: 0
 
 requirements:
-  host:
+  build:
     - python >=3.8
+    - scikit-build-core
```

Comment:
You need to list cmake here too. I can't make scikit-build-core depend on things like cmake and make due to issues with how conda works. :(